lib/verify.js

```diff
@@ -57,19 +58,30 @@ export async function verify({credential, capability, variables = {}} = {}) {
   };
 }
 
-export async function verifyVcb({text, getVerifyOptions} = {}) {
+export async function verifyVcb({data, format, getVerifyOptions} = {}) {
   try {
-    const verifyOptions = await getVerifyOptions({text});
+    const verifyOptions = await getVerifyOptions({
+      text: data,
+      data,
+      format
+    });
     const {documentLoader, verifyCredential} = verifyOptions;
     assert.func(documentLoader);
     assert.func(verifyCredential);
 
-    const {jsonldDocument: credential} = await util.fromQrCode({
-      text,
-      documentLoader,
-      typeTableLoader,
-      expectedHeader: 'VC1-',
-    });
+    let credential;
+
+    if(format === BARCODE_FORMATS.QR_CODE) {
+      const {jsonldDocument} = await util.fromQrCode({
+        text: data,
+        documentLoader,
+        typeTableLoader,
+        expectedHeader: 'VC1-',
+      });
+      credential = jsonldDocument;
+    } else {
+      throw new Error(`Barcode format, ${format}, not yet supported.`);
+    }
```

Comment:
Style nit: Do simple / quick / exceptional conditionals first, getting them out of the way for larger code blocks, enabling larger code blocks to have less indentation, and enabling use of `const` vars instead of `let`:

```suggestion
    if(format !== BARCODE_FORMATS.QR_CODE) {
      throw new Error(`Barcode format, ${format}, not yet supported.`);
    }
    
    const {jsonldDocument: credential} = await util.fromQrCode({
      text: data,
      documentLoader,
      typeTableLoader,
      expectedHeader: 'VC1-',
    });
```

Reply:
I'm thinking this will immediately extend to using pdf417s by using an if statement that says

```javascript
    let credential;
    let extraInformation;
    if(format === BARCODE_FORMATS.QR_CODE) {
      const {jsonldDocument} = await util.fromQrCode({
        text: data,
        documentLoader,
        typeTableLoader,
        expectedHeader: 'VC1-',
      });
      credential = jsonldDocument;

    } else if(format === BARCODE_FORMATS.PDF417) {
      const {jsonldDocument, hashed} = await parsePdf417({
        pdf417String: data,
        documentLoader,
        typeTableLoader,
      });
      credential = jsonldDocument;
      extraInformation = hashed;
    } else {
      throw new Error(`Barcode format, ${format}, not yet supported.`);
    }
```

so this is in anticipation of handling more barcode types